### ISSUE TITLE
Directories with a slash do result in a subdir

### ISSUE THREAD

**Issue by drewsed** [bruot/pyrmexplorer#18]
Hi,
thanks for your nice application. It's everyting I need to work without the cloud.

I have a directory like "Work / Life" and when I download all files I get two directories
- Work
  - Life

I know, that "/" isn't allowed as a name for a dir, but I would expect the "/" is somehow replaced with "-" for example.

**Comment by bruot** [comment]
Yes, there are issues with paths and filenames. See #12 for a similar problem.

To solve both issues, I think the best is to sanitize paths by allowing only a limited set of characters.
